test/integration/crud/bulk.test.js

```diff
@@ -21,6 +22,116 @@ describe('Bulk', function () {
     return setupDatabase(this.configuration);
   });
 
+  context('promise tests', () => {
+    it('Should correctly execute unordered bulk operation in promise form', {
+      metadata: {
+        requires: {
+          topology: ['single', 'replicaset', 'sharded']
```

Comment:
Do we actually want to explicitly exclude these from running against anything that returns "load-balanced" (same question on the other files where we have kept the metadata specified)?

Reply:
I wouldn't think so but I also wasn't sure.  I decided not to add the topology `load balanced` because even though I didn't think it would break anything, I wasn't sure.

This does make me think though - it seems like instead of opting-in to topologies, we should be explicitly blocking topologies.  I would guess that when Durran added `load-balanced` as an option, he didn't update all the tests to include `load-balanced` as a valid topology, which is why it's not included in this list.  Flipping the logic in our filter is definitely out of scope for this PR but I think it might be worthwhile.  Thoughts?

Reply:
But for now, for this particular test, I can remove the restrictions unless there's a reason not to.

Reply:
I think we should expect all our tests to work on all topologies by default, so the idea of opt out is sound, but out of scope here; we'd probably want to do something like "requires: { topology: { not: [ ... ] } }" for the tests that do need special filtering and think further about the implications; but yes, just removing the requirement for now on these tests is a good first step